inspirehep/base/templates/search/searchbar_frame_base.html

```diff
@@ -0,0 +1,34 @@
+{#
```

Comment:
I think it should work if the file is named `searchbar_frame.html` and then inside you extend `searchbar_frame_base.html` and just have as content:

```html
{% block body %}
   {% block inner_content %}{% endblock %}
{% endblock %}
```

Can you try?